### PR TITLE
Updated bundle image to latest tags and updated doc flow

### PR DESCRIPTION
**Pull request by dislbenn**
**Related Issue:**  stolostron/backlog#<ISSUE_NUMBER>

### Description of changes
Updated the bundled image doc flow and updated the images to the latest.

### CODE REVIEW

File: pg-snapshot.ver [hunk 1]
```diff
@@ -1 +1 @@
-1-56
\ No newline at end of file
+1-56
```
Comment (jlpadilla) [inline pg-snapshot.ver]: Let's use the most recent image `1-78`
https://catalog.redhat.com/software/containers/rhel8/postgresql-13/5ffdbdef73a65398111b8362

Comment (dislbenn) [reply pg-snapshot.ver]: Updated to the most recent image.

**Review by jlpadilla — APPROVED** [review]
/lgtm

**Pull request merged**